systest/cluster/nodes.go

```diff
@@ -188,12 +188,15 @@ func deployNodes(ctx *testcontext.Context, name string, from, to int, flags []De
 	)
 	for i := from; i < to; i++ {
 		i := i
-		finalFlags := make([]DeploymentFlag, len(flags)+1)
+		finalFlags := make([]DeploymentFlag, len(flags))
```

Comment:
If you want to avoid re-allocating the slice multiple times do this:

```suggestion
		finalFlags := make([]DeploymentFlag, len(flags), len(flags)+ctx.PoetSize)
```